rpa-home/modules/ROOT/pages/automation-usecase-legacysystemdataextraction-extractfrompdf.adoc

```diff
@@ -0,0 +1,64 @@
+# Finding Information in a PDF
+// number the headings to correspond to the numbered list of steps
+:sectnums:
+
+First, you need to find the order ID in the PDF.
+
+As you can see in xref:automation-usecase-legacysystemdataextraction.adoc#confirmationletter-image[the image of the order confirmation letter], the order ID is always listed in a table next to the entry 'Order ID'. 
+// Do we need an image of that part of the letter here?
+You can use this information to implement the workflow that contains the following steps:
+
+. Copy the content of the order confirmation to a text file.
+. Read the text file line by line to find the one containing the order ID.
+. Check if the order ID is found
+. Cut the order ID out from the text and store it in an activity parameter.
+
+image::usecase-legacysystem-extractformpdf-workflow-overview.png["The Workflow contains the steps in form of Action Steps. Some of the Action Steps are nested."]
+
+## Copying Text From a PDF to a Text File
+To open the PDF, you use *Read PDF file*. Then you use the returned `ReadText` as input for *Write to Text File* to save the content as text file on your computer.
+
+image::usecase-legacysystem-extractformpdf-workflow-pdftotxt.png["An out-filled pin symbol next to the input field shows that the variable is mapped to the text extracted from the PDF in a previous Action Step."]
```

Comment:
I'm not sure "out-filled pin symbol" is obvious. Suggest something like "A variable mapped to the text extracted...."

Reply:
That is the additional information that seeing readers can get from the screenshot.